helix-term/src/keymap.rs

```diff
@@ -62,252 +283,298 @@ impl DerefMut for Keymaps {
 
 impl Default for Keymaps {
     fn default() -> Keymaps {
-        let normal = hashmap!(
-            key!('h') => Command::move_char_left,
-            key!('j') => Command::move_line_down,
-            key!('k') => Command::move_line_up,
-            key!('l') => Command::move_char_right,
-
-            key!(Left) => Command::move_char_left,
-            key!(Down) => Command::move_line_down,
-            key!(Up) => Command::move_line_up,
-            key!(Right) => Command::move_char_right,
-
-            key!('t') => Command::find_till_char,
-            key!('f') => Command::find_next_char,
-            key!('T') => Command::till_prev_char,
-            key!('F') => Command::find_prev_char,
-            // and matching set for select mode (extend)
-            //
-            key!('r') => Command::replace,
-            key!('R') => Command::replace_with_yanked,
-
-            key!('~') => Command::switch_case,
-            alt!('`') => Command::switch_to_uppercase,
-            key!('`') => Command::switch_to_lowercase,
-
-            key!(Home) => Command::goto_line_start,
-            key!(End) => Command::goto_line_end,
-
-            key!('w') => Command::move_next_word_start,
-            key!('b') => Command::move_prev_word_start,
-            key!('e') => Command::move_next_word_end,
-
-            key!('W') => Command::move_next_long_word_start,
-            key!('B') => Command::move_prev_long_word_start,
-            key!('E') => Command::move_next_long_word_end,
-
-            key!('v') => Command::select_mode,
-            key!('g') => Command::goto_mode,
-            key!(':') => Command::command_mode,
-
-            key!('i') => Command::insert_mode,
-            key!('I') => Command::prepend_to_line,
-            key!('a') => Command::append_mode,
-            key!('A') => Command::append_to_line,
-            key!('o') => Command::open_below,
-            key!('O') => Command::open_above,
+        let normal = keymap!({ "Normal mode"
```

Comment:
Can keymap be `keymap! {`?

Reply:
No, since the argument has to be a token tree so that it can recurse properly and detect sub nodes.